jenkins-agent.pkr.hcl

```diff
@@ -183,7 +188,25 @@ source "azure-arm" "base" {
   }
 }
 
+source "docker" "base" {
+  # only amd64 supported
+  image = "ubuntu:20.04"
+  # persist image on local docker engine
+  commit = true
+  # some label to stay compliant
```

Comment:
```suggestion
  # some labels to stay compliant
```

Reply:
Commiting this suggestion should be enough to trigger the build as I've added you to a team with rights here.